### ISSUE TITLE
Unable to clone VM

### ISSUE THREAD

**Issue by theobromina** [chef-partners/knife-vsphere#208]
When trying to clone a VM I get the following error:

C:\Windows\system32>knife vsphere vm clone CHEFLAB001
Cloning template CHEFUBUNTU1204 to new VM CHEFLAB001
ERROR: RbVmomi::Fault: CannotAccessVmConfig: Unable to access the virtual machine configuration: Unable to access file [Host-DS] CHEFUBUNTU1204/CHEFUBUNTU1204.vmx

I specified the source vm in knife.rb
Made sure user had full rights, even used administrator@vsphere.local

Running the following version of chef:

C:\Windows\system32>chef -v
Chef Development Kit Version: 0.9.0
chef-client version: 12.5.1
berks version: 4.0.1
kitchen version: 1.4.2

Any help would be greatly appreciated.

**Comment by adamleff** [comment]
@theobromina I'm not sure what this could be within the knife-vsphere gem itself as the error is coming back from vCenter / the RbVmomi API gem.

Presuming that your source VM is named "CHEFUBUNTU1204", can you perform a state command on it to ensure the API can find your VM to begin with?

example: `knife vsphere vm state CHEFUBUNTU1204`

Also, is CHEFUBUNTU1204 in a folder, or is it in the base folder of your datacenter?

**Comment by swalberg** [comment]
Can you paste your `knife.rb` please (with sensitive stuff X'ed out)

**Comment by theobromina** [comment]
@swalberg Here it is:
log_level                :info
log_location             STDOUT
node_name                'Admin_name'
client_key               'C:/Users/Admin_name.DOMAIN/.chef/Admin_name.pem'
validation_client_name   'chef-validator'
validation_key           'C:/etc/chef-server/chef-validator.pem'
chef_server_url          'https://WORKSTATION.DOMAIN.COM:443'
syntax_check_cache_path  'C:/Users/Admin_name.DOMAIN/.chef/syntax_check_cache'
knife[:vsphere_host] = "vCenter_Server.DOMAIN.COM"
knife[:vsphere_user] = "Administrator@vsphere.local"
knife[:source_vm] = "CHEFUBUNTU1204"
knife[:vsphere_pass] = "[password]"
knife[:vsphere_dc] = "[datacenter]"
knife[:vsphere_insecure] = true

**Comment by theobromina** [comment]
@adamleff It returns: 
>knife vsphere vm state CHEFUBUNTU1204
VM CHEFUBUNTU1204 is currently powered off

**Comment by theobromina** [comment]
Hi all!
Was able to fix this problem by doing the following:

1. Made sure the folder containing the templates was available to every host under the vCenter server.
2. Specified –resource-pool. With this option I designated the cluster the VM would be deployed to.
3. Once I got passed that error, there was a new one (NIC settings mismatch) that was resolved by specifying –cspec and –cips

Thanks for the assistance @swalberg and @adamleff  Your questions pointed me in the right direction.
